src/components/accordion/index.md.njk

```diff
@@ -17,83 +17,119 @@ The accordion component lets users show and hide sections of related content on
 
 ### Do not disable sections
 
 Disabling controls is normally confusing for users. If there is no content for a section, either remove the section or, if this would be confusing for your users, explain why there is no content when the section is opened.
 
 ## Research on this component
 
-This component is experimental because there are different approaches to accordions in services. More research is needed to determine the best approach, or to know when a different approach works better.
+We updated this component in November 2021 to solve an accessibility issue where the buttons and section labels might be mistaken for links.
 
-The design for this component was originally created and tested by a team at the Government Digital Service (GDS) on the [GOV.UK Service Manual](https://www.gov.uk/service-manual) and [GOV.UK topic pages](https://designnotes.blog.gov.uk/2017/06/29/designing-new-navigation-elements-for-gov-uk/).
+[Read about the research and development that went into improving the accordion component](https://github.com/alphagov/govuk-design-system-backlog/issues/1).
 
 The team made sure the component is accessible, for example that users can interact with it using just the keyboard.
 
-### Known issues and gaps
+#### Users that navigate using ‘elements lists’
+
+We need to find out more about users that navigate using ‘elements lists’ of headings, buttons, links and other elements — such as users of speech recognition software and partially-sighted users of screen readers.
 
-The section headings can be mistaken for links, but are treated as buttons. This fails [WCAG 2.1 success criterion 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships). We plan to fix this issue by the end of June 2021.
+For these users, it might not be clear enough that the section headings are considered buttons, misleading them to navigate (less efficiently) using the show or hide labels.
 
-The plus and minus icon is on the right side of the component, which means users of screen magnifiers might not see it.
+While inconvenient at first, it’s likely the user will better understand the button area once they interact with it and see its focus state. [See ‘Testing with Dragon’ in the accessibility clinic summary](https://github.com/alphagov/govuk-frontend/issues/2295#issuecomment-906449543).
 
-The 'Open all' button reads out as 'Open all sections' for screen readers. This is potentially confusing for users as the visual content is different to what screen readers read out.
+We want to hear about any user research done in this area so we can identify potential issues.
+
+### Known issues and gaps
 
-### Next steps
-Investigate the problem of some users not seeing the plus and minus icons on the right, for example people using screen magnifiers.
+Adding a summary line with more than a few short words will likely make the button text too long, particularly for users of x readers.```
```

Comment:
Looks like this sentence contains a place holder. @calvin-lau-sig7 Let me know if I can help with wrapping this up.

Reply:
Thanks both — mushy brian on my part.

```suggestion
Adding a summary line with more than a few short words will likely make the button text too long, particularly for users of screen readers.```